docs/components/badge.md

```diff
@@ -77,32 +80,40 @@ Using the `custom` appearance in conjunction with `color` and `background-color`
 | `--KBadgeDangerBorder`      |                               |
 | `--KBadgeDangerBackground`  |                               |
 
-An example of making the badges more like pills
+An example of making the badges more rad
 
 > Note: We are scoping the overrides to a wrapper in this example
 
 <template>
   <div class="KBadge-wrapper">
-    <KBadge appearance="danger">HIGH</KBadge>
+    <KBadge appearance="danger">DANGER - RADIOACTIVE MATERIAL</KBadge>
   </div>
 </template>
 
 ```vue
 <template>
   <div class="KBadge-wrapper">
-    <KBadge appearance="danger">HIGH</KBadge>
+    <KBadge appearance="danger">DANGER</KBadge>
```

Comment:
Should say `DANGER - RADIOACTIVE Material`

Reply:
![image](https://user-images.githubusercontent.com/67973710/107409426-27ea1880-6ada-11eb-9be7-2b11d6006668.png)

Reply:
Fixed in https://github.com/Kong/kongponents/pull/288/commits/e60c754e076146508d4c910a8cd775f7b8841dd9